client/tests/conftest.py

```diff
@@ -7,8 +7,11 @@
 API_URL = os.getenv("API_URL", "http://localhost:5050/api/v1/")
 SUPERADMIN_LOGIN = os.getenv("SUPERADMIN_LOGIN", "superadmin_login")
 SUPERADMIN_PWD = os.getenv("SUPERADMIN_PWD", "superadmin_pwd")
+SUPERADMIN_ORGANIZATION = os.getenv("SUPERADMIN_ORGANIZATION", "superadmin_organization")
```

Comment:
I'd suggest organization --> org everywhere for the env variable to keep it short